openquake/hazardlib/const.py

```diff
@@ -105,5 +105,9 @@ class StdDev(object):
     ALL = 'All'
     #: Compute all the standard deviations for which the GMPE is defined
 
+    SINGLE_STATION = 'Single station'
+    #: Standard deviation representing ground shaking variability
+    #: within a single site.
+
```

Comment:
It looks like the constant SINGLE_STATION is never used.

Reply:
Fixed